FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,7 +1,6 @@
-# Unversioned
+# v4.2.0
+- [Added] Plist property"FirebaseDeepLinkPasteBoardRetrievalEnabled" to enable/disable fetching dynamic links from PasteBoard.
```

Comment:
Organization looks good.

From my read of docs, the "B" in pasteboard should be lowercase here and in the implementation.

Use a  backtick ``` instead of `"` around the property name.

Reply:
Made changes...Please check whether I'm using the right one for "backtick"